cmd/formula-analytics.rb

```diff
@@ -355,7 +355,14 @@ def influx_analytics(args)
 
     odie "No InfluxDB credentials found in HOMEBREW_INFLUXDB_TOKEN!" unless ENV["HOMEBREW_INFLUXDB_TOKEN"]
 
-    puts "TODO: use client: #{influxdb_client}"
+    query = <<~EOS
+      from(bucket: "analytics")
+        |> range(start: -1m, stop: now())
+        |> filter(fn: (r) => r._measurement == "formula_install")
+        |> group(columns: ["package_and_options"])
```

Comment:
This query will give you the data you want. I just can't get InfluxDB to do the sorting right.
```suggestion
      from(bucket: "analytics")
        |> range(start: -31d, stop: now())
        |> filter(fn: (r) => r["_measurement"] == "formula_install")
        |> group(columns: ["package_and_options"])
        |> sum()
```

Reply:
@SMillerDev nice, thanks! Can always do sorting in Ruby 🤷🏻